src/wasm/wasm-validator.cpp

```diff
@@ -2143,6 +2144,66 @@ void FunctionValidator::visitTry(Try* curr) {
                 curr,
                 "try cannot have both catch and delegate at the same time");
 
+  // Given a catch body, find pops corresponding to the catch
+  auto findPops = [](Expression* expr) {
+    std::vector<Pop*> pops;
+    std::vector<Expression*> work;
```

Comment:
These could be SmallVectors perhaps? But probably the difference wouldn't be noticeable.

Reply:
Done